README.md

```diff
@@ -129,7 +131,14 @@ export const ChatScreen: React.FC = () => {
         onPressGallery,
       }}
     >
-    {({onSend}) => (<CameraView onSend={onSend} /> )}
+    {({onSend}) => (
```

Comment:
did you forget to guide on how to show the document button on InputToolBar?

Reply:
yes, I added